dashboard/src/pages/schedule/Schedule.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <Header />
+  <div
+    v-if="event.data && schedule.data"
+    class="w-full flex flex-col items-center p-4"
+  >
+    <div class="max-w-screen-xl w-full">
+      <Breadcrumb class="mt-2" :items="breadcrumb_items" />
```

Comment:
The redirect back to events page isnt working as expected:

![route-schedule-test-2](https://github.com/user-attachments/assets/c6d945e6-392a-45c6-8af3-66f7e9ab5d02)

Reply:
Port 8000 btw

[Screencast from 2024-08-28 15-41-12.webm](https://github.com/user-attachments/assets/dbb4bab8-cdc5-4238-aa63-e64beda5f126)